### PR TITLE
Cleanup configuration

### PR DESCRIPTION
**Pull request by brtkwr**
…vars

### CODE REVIEW

File: config/slurm.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+cluster_name: "slurm"
+cluster_num_workers: 3
+storage_name: bharat-nfs
```
Comment (oneswig) [inline config/slurm.yml]: Bharat detector went bzzt...

File: group_vars/all/k8s [hunk 2]
```diff
@@ -1,4 +1,3 @@
 ---
 # Cloud provider to use for Kubernetes clusters
 cloud_provider: openstack
-...
```
Comment (oneswig) [inline group_vars/all/k8s:4]: outrageous

File: playbooks/nfs.yml [hunk 3]
```diff
@@ -38,3 +38,4 @@
         clients:  "{{ 'nfs_clients' in groups and inventory_hostname in groups['nfs_clients'] }}" 
       nfs_server: "{{ groups['nfs_servers'] | first }}"
       nfs_export: "{{ hostvars[groups['nfs_servers'] | first]['nfs_export'] }}"
+      nfs_client_mnt_point: "{{ nfs_client_mnt_point }}"
```
Comment (oneswig) [inline playbooks/nfs.yml]: There's risks about making this variable - lots of things (eg, user home dir paths) depend on it.  Just saying.

**Review by oneswig — APPROVED** [review]
Couple of minor nits here but shouldn't hold back progress.

File: group_vars/all/k8s (hunk 2 above)
Comment (brtkwr) [reply group_vars/all/k8s:4]: I have decided to abandon the practice. Its a lot of pain for little gain.

File: playbooks/nfs.yml (hunk 3 above)
Comment (brtkwr) [reply playbooks/nfs.yml]: It now points to "jasmin_homedir" instead

**Pull request merged**